### ISSUE TITLE
reflectable package incompatible to flutter

### ISSUE THREAD

**Issue by BuhTigRunner** [dart-lang/reflectable#127]
Please see the link
[https://github.com/flutter/flutter/issues/16590](https://github.com/flutter/flutter/issues/16590)
It possible to create a new version that is compatible with later versions of the path.Is it possible to create a new version that is compatible with later versions of the path?

**Comment by eernstg** [comment]
I'm waiting for a review (and the rest of this week consists of holidays here in Denmark), but you might be able to use the branch `version_constraint_update_may18` from github:

```
dependencies:
  ...
  reflectable:
    git: 
      url: git://github.com/dart-lang/reflectable.git
      ref: version_constraint_update_may18
```

**Comment by eernstg** [comment]
Some vacation days and trips abroad later, I finally got this published: Version 2.0.1 on `pub.dartlang.org` is the result of merging `version_constraint_update_may18`. If you have used the above `ref:` you should be able to just use `reflectable: ^2.0.0` and then you'll get `2.0.1` and up automatically.

**Comment by eernstg** [comment]
I'll close this issue now, please open another one if this doesn't resolve the version conflict that you mentioned.
